tests/config/install.sh

```diff
@@ -44,6 +44,7 @@ ln -sf $SRC_PATH/config.d/logger.xml $DEST_SERVER_PATH/config.d/
 ln -sf $SRC_PATH/config.d/named_collection.xml $DEST_SERVER_PATH/config.d/
 ln -sf $SRC_PATH/config.d/ssl_certs.xml $DEST_SERVER_PATH/config.d/
 ln -sf $SRC_PATH/config.d/filesystem_cache_log.xml $DEST_SERVER_PATH/config.d/
+ln -sf $SRC_PATH/config.d/local_disks.xml $DEST_SERVER_PATH/config.d/
```

Comment:
If we need this only for one test it's better to make it integration test with custom config

Reply:
Sorry, I am not so familiar with current test workflow, is there any example to define a storage policy in custum config

Reply:
For example https://github.com/ClickHouse/ClickHouse/tree/master/tests/integration/test_multiple_disks